### PR TITLE
[UI]: Update the Outdated Button Component to Sistent and updated import

### PR DESCRIPTION
**Pull request by Ajay-singh1**
**Notes for Reviewers**

Updated the outdated button component to Sistent , I have tested it and verified it on my local machine.The button was using the default theme provided by Material UI I have updated it to use Sistent Theme for better consistency of the user interface.

Screenshot from 2024-11-17 15-19-29
- This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@Ajay-singh1 
Let's discuss this during the Meshery Development call today at 9:00 AM CT | 8:30 PM IST. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#) || [meeting link](https://meet.layer5.io/meshery).

File: ui/components/General/ErrorBoundary.js [hunk 1]
```diff
@@ -20,14 +21,16 @@ function Fallback({ error, resetErrorBoundary }) {
...
       </div>
-      <Button
-        color="primary"
-        variant="contained"
-        onClick={resetErrorBoundary}
-        data-testid={generateTestIDs('button')}
-      >
-        Try again
-      </Button>
+      <SistentThemeProvider>
```
Comment (sudhanshutech) [inline ui/components/General/ErrorBoundary.js:24]: use the wrapper usesSistent not this because its use with wrapper already

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by Ajay-singh1** [comment]
@sudhanshutech  This PR is ready to be merged now.

**Comment by leecalcote** [comment]
@sudhanshutech we're ready for final review here.

**Pull request closed without merging**